rs/ethereum/cketh/playground/README.adoc

```diff
@@ -97,7 +97,7 @@ If you deploy from a dev env,
 [source,shell]
 ----
 git checkout 5553ee5f7586290a59ed372a2b7aca847520af82
-./gitlab-ci/container/build-ic.sh -c
```

Comment:
Since the line above references an exact commit, I would not change the instructions here.